functions.php

```diff
@@ -17,6 +17,16 @@
 	define( 'NEVE_DEBUG', false );
 }
 define( 'NEVE_NEW_DYNAMIC_STYLE', true );
+
+/**
+ * Flag to decide if the 'skip-lazy' class should be added in post loop.
+ *
+ * @var bool $neve_thumbnail_skip_lazy_added
+ */
+global $neve_thumbnail_skip_lazy_added;
```

Comment:
You don't need this here, just add this in ` inc/views/template_parts.php`
```php
global $neve_thumbnail_skip_lazy_added;
if( isset($neve_thumbnail_skip_lazy_added) )  
```
to check if you should add/not the class.